### DISCUSSION TITLE
how to solve 'Invalid connection url for multiconnection'

### DISCUSSION THREAD

**Discussion by holicc** [diesel-rs/diesel#3685] [Q&A]
postgresql version : 14.7 

env file: 

`DATABASE_URL=postgres://user:pass@localhost:5432/db`

**Comment by BanjoFox** [comment]
Was this something that was ever *fixed* or simply closed out?
I'm seeing the error while trying to configure Github Actions.  

The database, and user appear to be created successfully, but I keep hitting the error
```
Run diesel migration run
  diesel migration run
  shell: /usr/bin/bash -e {0}
  env:
    CARGO_INCREMENTAL: 0
    CARGO_TERM_COLOR: always
    DATABASE_URL: postgres://aardwolf_user@localhost:543[2](https://github.com/Aardwolf-Social/aardwolf/actions/runs/5764782857/job/15629460555#step:7:2)/aardwolf_testing
Could not connect to database via `postgres://aardwolf_user@localhost:54[3](https://github.com/Aardwolf-Social/aardwolf/actions/runs/5764782857/job/15629460555#step:7:3)2/aardwolf_testing`: Invalid connection url for multiconnection
Error: Process completed with exit code 1.
```

Setting the ENV using this:

```
   - run: echo 'DATABASE_URL=postgres://aardwolf_user:changeme@localhost/aardwolf_testing' > "$GITHUB_ENV"
```

**Reply by weiznich** [reply]
There is nothing to fix in diesel here as this is not a bug with diesel, but a problem with your setup. This error message just indicates that it is not possible to connect to any database using that url.

**Reply by BanjoFox** [reply]
That is what I figured, but I am also having trouble figuring out how to fix my environment.  Will keep searching.
